docs/src/man/guide.md

```diff
@@ -124,10 +124,43 @@ build/
         documenter.js
         search.js
     index.html
-    search.html
+    search/index.html
     search_index.js
 ```
 
+!!! note
+
+    By default, Documenter has pretty URLs enabled, which means that `src/foo.md` is turned
+    into `src/foo/index.html`, instead of simply `src/foo.html`, which is the preferred way
+    when creating a set of HTML to be hosted on a web server.
+
+    However, this can be a hindrance when browsing the documentation locally as browsers do
+    not resolve directory URLs like `foo/` to `foo/index.html` for local files. You have two
+    options:
+
+    1. You can run a local web server out of the `docs/build` directory. If you have Python
+       installed, you can simple start one with `python3 -m http.server --bind localhost`
```

Comment:
Isn't there a way to do the same with Julia?
https://github.com/JuliaWeb/HttpServer.jl/issues/143

Reply:
Most likely and I would happily take a PR that documents such a workflow. But Python seems to be the easiest option at the moment since it's literally just one command on the command line, and it's what I happen to use.

Also, a more sophisticated implementation, more in the vein of `mkdocs serve`, could be implemented in DocumenterTools (JuliaDocs/DocumenterTools.jl#18).